cmd/apprepository-controller/cmd/root.go

```diff
@@ -32,6 +32,7 @@ func newRootCmd() *cobra.Command {
 		Use:   "apprepository-controller",
 		Short: "Apprepository-controller is a Kubernetes controller for managing package repositories added to Kubeapps.",
 		PreRun: func(cmd *cobra.Command, args []string) {
+			initServerOpts()
```

Comment:
Great! Thanks for the fix! Haven't checked, but I wonder whether this same issue also appears in the rest of the cobra cmd we have in the project (kubeops, kubeapps-apis, etc...)

Reply:
Yep, at least in Kubeops. I'll check the others and do a separate diff.